seatunnel-examples/seatunnel-flink-examples/src/main/resources/examples/fake_to_console.config

```diff
@@ -0,0 +1,39 @@
+######
+###### This config file is a demonstration of streaming processing in seatunnel config
+######
+
+env {
+  # You can set flink configuration here
+  execution.parallelism = 1
+  #execution.checkpoint.interval = 10000
+  #execution.checkpoint.data-uri = "hdfs://localhost:9000/checkpoint"
+}
+
+source {
+  # This is a example source plugin **only for test and demonstrate the feature source plugin**
+    FakeSourceStream {
+      result_table_name = "fake"
+      field_name = "name,age"
+    }
+
+  # If you would like to get more information about how to configure seatunnel and see full list of source plugins,
+  # please go to https://interestinglab.github.io/seatunnel-docs/#/zh-cn/configuration/base
+}
+
+transform {
+    sql {
+      sql = "select name,age from fake"
+    }
+
+  # If you would like to get more information about how to configure seatunnel and see full list of transform plugins,
+  # please go to https://interestinglab.github.io/seatunnel-docs/#/zh-cn/configuration/base
+}
+
+sink {
+  ConsoleSink {}
+
+
+  # If you would like to get more information about how to configure seatunnel and see full list of sink plugins,
+  # please go to https://interestinglab.github.io/seatunnel-docs/#/zh-cn/configuration/base
+}
```

Comment:
Do you want to enable the new official document address for the document address here?

Reply:
Let me change it. I copied it from the demo. I forgot to change it